lib/reportportal-agent.js

```diff
@@ -9,8 +9,9 @@ class ReportportalAgent {
             endpoint: '',
             project: '',
         }, conf);
+        const agentParams = SpecificUtils.getAgentParams();
         const launchObj = SpecificUtils.getLaunchObj(conf);
-        this.client = new RPClient(config);
+        this.client = new RPClient(config, agentParams);
```

Comment:
May be rename `agentParams` to smth like `agentInfo`? Just because config is agent params too.